pyat/at/lattice/elements.py

```diff
@@ -972,6 +972,45 @@ def __init__(self, family_name: str, m66=None, **kwargs):
         super(M66, self).__init__(family_name, M66=m66, **kwargs)
 
 
+class SimpleQuantDiff(Element):
+    """Linear tracking element for simplified quantum diffusion"""
+    _BUILD_ATTRIBUTES = Element._BUILD_ATTRIBUTES + ['emit_x',
+                                                     'emit_y',
+                                                     'sigma_dp',
+                                                     'tau_x',
+                                                     'tau_y',
+                                                     'tau_z',
+                                                     'beta_x',
+                                                     'beta_y',
+                                                     'U0']
+
+    def __init__(self, family_name: str, emit_x: float, emit_y: float,
+                 sigma_dp: float, tau_x: float, tau_y: float, tau_z: float,
+                 beta_x: float, beta_y: float, U0: float, **kwargs):
+        """
+        Args:
+            family_name:    Name of the element
+            emit_x:         Horizontal equilibrium emittance [m.rad]
+            emit_y:         Vertical equilibrium emittance [m.rad]
+            sigma_dp:       Equilibrium energy spread
+            tau_x:          Horizontal damping time [turns]
+            tau_y:          Vertical damping time [turns]
+            tau_z:          Longitudinal damping time [turns]
+            beta_x:         Horizontal beta function at element [m]
+            beta_y:         Vertical beta function at element [m]
+            U0:             Energy Loss [eV]
+            
+        Default PassMethod: ``SimpleQuantDiffPass``
+       """
+        kwargs.setdefault('PassMethod', 'SimpleQuantDiffPass')
+        super(SimpleQuantDiff, self).__init__(family_name,
```

Comment:
Why pass all the specific arguments to your supper method?
You may instead declare the element's attribute in the __init__ method as:

```
if emitx is not None:
    self.emitx=emitx
```

etc...